packages/bits/spec/directives/resizer/resizer.atom.ts

```diff
@@ -20,7 +20,7 @@ export class ResizerAtom extends Atom {
     public getResizeDirection = async (): Promise<"horizontal" | "vertical"> => {
         const className = await this.resizeSplitElement.getAttribute("class");
         return className.includes("horizontal") ? "horizontal" : "vertical";
-    }
+    };
```

Comment:
This may fail linting. I'm not sure, but I believe there's a rule for no semicolons at the end of lambda definitions.

Reply:
remove semocolons